doc/nimc.md

```diff
@@ -316,14 +316,16 @@ Another way is to make Nim invoke a cross compiler toolchain:
   nim c --cpu:arm --os:linux myproject.nim
   ```
 
-For cross compilation, the compiler invokes a C compiler named
-like `$cpu.$os.$cc` (for example arm.linux.gcc) and the configuration
-system is used to provide meaningful defaults. For example for `ARM` your
+For cross compilation, the compiler invokes a C compiler named like
+`$cpu.$os.$cc` (for example `arm.linux.gcc`) with options defined in
+`$cpu.$os.$cc.options.always`. The configuration system is used to provide
+meaningful defaults. For example, for Linux on a 32-bit ARM CPU, your
 configuration file should contain something like:
 
     arm.linux.gcc.path = "/usr/bin"
     arm.linux.gcc.exe = "arm-linux-gcc"
     arm.linux.gcc.linkerexe = "arm-linux-gcc"
+    arm.linux.gcc.options.always = "-w -fmax-errors=3"
```

Comment:
What does `-w` do? And why `-fmax-errors=3`?

Reply:
`-w` suppresses warning, and TBH I'm not even sure about the exact behavior of `fmax-errors`. I put in these parameters because they appear to be the common subset for gcc compilation of nim on all platforms. See:

https://github.com/nim-lang/Nim/blob/devel/config/nim.cfg#L192

(`${gcc.maxerrorsimpl}` resolves to `fmax-erorrs=3` somewhere in options.nim.